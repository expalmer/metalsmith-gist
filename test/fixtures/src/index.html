---
title: test
gist: expalmer/4bc6203dc2a3f7134e78 sukima/510438:load_config.rb
---

Hello World

gist:expalmer/4bc6203dc2a3f7134e78

gist:sukima/510438:load_config.rb
