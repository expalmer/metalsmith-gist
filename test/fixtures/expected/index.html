
Hello World

<link rel="stylesheet" href="https://gist-assets.github.com/assets/embed-506ce4903ba8030610b1aacf371fd7b0.css"><div id="gist13951371" class="gist">        <div class="gist-file">          <div class="gist-data gist-syntax">                <div class="file-data">      <table cellpadding="0" cellspacing="0" class="lines highlight">        <tr>          <td class="line-numbers">            <span class="line-number" id="file-gistfile1-js-L1" rel="file-gistfile1-js-L1">1</span>            <span class="line-number" id="file-gistfile1-js-L2" rel="file-gistfile1-js-L2">2</span>            <span class="line-number" id="file-gistfile1-js-L3" rel="file-gistfile1-js-L3">3</span>            <span class="line-number" id="file-gistfile1-js-L4" rel="file-gistfile1-js-L4">4</span>            <span class="line-number" id="file-gistfile1-js-L5" rel="file-gistfile1-js-L5">5</span>            <span class="line-number" id="file-gistfile1-js-L6" rel="file-gistfile1-js-L6">6</span>            <span class="line-number" id="file-gistfile1-js-L7" rel="file-gistfile1-js-L7">7</span>            <span class="line-number" id="file-gistfile1-js-L8" rel="file-gistfile1-js-L8">8</span>          </td>          <td class="line-data">            <pre class="line-pre"><div class="line" id="file-gistfile1-js-LC1"><span class="kd">var</span> <span class="nx">Animal</span> <span class="o">=</span> <span class="kd">function</span> <span class="p">(</span> <span class="nx">name</span> <span class="p">)</span> <span class="p">{</span></div><div class="line" id="file-gistfile1-js-LC2">  <span class="k">this</span><span class="p">.</span><span class="nx">name</span> <span class="o">=</span> <span class="nx">name</span><span class="p">;</span></div><div class="line" id="file-gistfile1-js-LC3"><span class="p">}</span></div><div class="line" id="file-gistfile1-js-LC4"><span class="nx">Animal</span><span class="p">.</span><span class="nx">prototype</span><span class="p">.</span><span class="nx">getName</span> <span class="o">=</span> <span class="kd">function</span> <span class="p">()</span> <span class="p">{</span></div><div class="line" id="file-gistfile1-js-LC5">  <span class="k">return</span> <span class="k">this</span><span class="p">.</span><span class="nx">name</span><span class="p">;</span></div><div class="line" id="file-gistfile1-js-LC6"><span class="p">}</span></div><div class="line" id="file-gistfile1-js-LC7"><span class="kd">var</span> <span class="nx">tiger</span> <span class="o">=</span> <span class="k">new</span> <span class="nx">Animal</span><span class="p">(</span><span class="s1">&#39;Tiger&#39;</span><span class="p">);</span></div><div class="line" id="file-gistfile1-js-LC8"><span class="nx">tiger</span><span class="p">.</span><span class="nx">getName</span><span class="p">();</span></div></pre>          </td>        </tr>      </table>    </div>          </div>          <div class="gist-meta">            <a href="https://gist.github.com/expalmer/4bc6203dc2a3f7134e78/raw/gistfile1.js" style="float:right">view raw</a>            <a href="https://gist.github.com/expalmer/4bc6203dc2a3f7134e78#file-gistfile1-js">gistfile1.js</a>            hosted with &#10084; by <a href="https://github.com">GitHub</a>          </div>        </div></div>

<link rel="stylesheet" href="https://gist-assets.github.com/assets/embed-506ce4903ba8030610b1aacf371fd7b0.css"><div id="gist510438" class="gist">        <div class="gist-file">          <div class="gist-data gist-syntax">                <div class="file-data">      <table cellpadding="0" cellspacing="0" class="lines highlight">        <tr>          <td class="line-numbers">            <span class="line-number" id="file-load_config-rb-L1" rel="file-load_config-rb-L1">1</span>            <span class="line-number" id="file-load_config-rb-L2" rel="file-load_config-rb-L2">2</span>            <span class="line-number" id="file-load_config-rb-L3" rel="file-load_config-rb-L3">3</span>            <span class="line-number" id="file-load_config-rb-L4" rel="file-load_config-rb-L4">4</span>            <span class="line-number" id="file-load_config-rb-L5" rel="file-load_config-rb-L5">5</span>            <span class="line-number" id="file-load_config-rb-L6" rel="file-load_config-rb-L6">6</span>            <span class="line-number" id="file-load_config-rb-L7" rel="file-load_config-rb-L7">7</span>          </td>          <td class="line-data">            <pre class="line-pre"><div class="line" id="file-load_config-rb-LC1"><span class="c1"># Uses local config via YAML.</span></div><div class="line" id="file-load_config-rb-LC2"><span class="c1"># http://gist.github.com/510438</span></div><div class="line" id="file-load_config-rb-LC3">&nbsp;</div><div class="line" id="file-load_config-rb-LC4"><span class="no">APP_CONFIG</span> <span class="o">=</span> <span class="no">YAML</span><span class="o">.</span><span class="n">load_file</span><span class="p">(</span><span class="s2">&quot;</span><span class="si">#{</span><span class="no">RAILS_ROOT</span><span class="si">}</span><span class="s2">/config/config.yml&quot;</span><span class="p">)</span></div><div class="line" id="file-load_config-rb-LC5"><span class="no">APP_CONFIG</span><span class="o">.</span><span class="n">merge!</span><span class="p">(</span><span class="no">APP_CONFIG</span><span class="o">[</span><span class="s1">&#39;environment&#39;</span><span class="o">][</span><span class="no">RAILS_ENV</span><span class="o">]</span><span class="p">)</span></div><div class="line" id="file-load_config-rb-LC6"><span class="no">APP_CONFIG</span><span class="o">.</span><span class="n">delete</span><span class="p">(</span><span class="s1">&#39;environment&#39;</span><span class="p">)</span></div><div class="line" id="file-load_config-rb-LC7"><span class="no">APP_CONFIG</span><span class="o">.</span><span class="n">symbolize_keys</span></div></pre>          </td>        </tr>      </table>    </div>          </div>          <div class="gist-meta">            <a href="https://gist.github.com/sukima/510438/raw/load_config.rb" style="float:right">view raw</a>            <a href="https://gist.github.com/sukima/510438#file-load_config-rb">load_config.rb</a>            hosted with &#10084; by <a href="https://github.com">GitHub</a>          </div>        </div></div>
